get_started_with_gx_cloud/workshop.md

```diff
@@ -175,15 +175,13 @@ GX Cloud provides a tool to generate SQL based on a prompt. The prompt accepts p
 > 1. Click on the **Expectations** tab.
 > 1. Click **+ New Expectation**.
 > 1. Click the **SQL** at the bottom of the panel.
-> 1. In the text area, enter in `Check that rides from Queens to EWR have a fare amount of at least 75`.
+> 1. In the `Prompt for SQL generation` text area, enter in `Check that rides from Queens to EWR have a fare amount of at least 75`.
 > 1. Click **Generate SQL**
 > Click **Save**.
 
-<img src="img/generate_new_sql_expectation.png" alt="Generate a custom sql expectation" style="width:500px;"/><br>
-
 In this example, we're selecting all rides from `Queens` to the Newark Airport (code `EWR`), where the fares are less than `75` ($75 US). We know that Queens to EWR is generally a long drive, so we expect it to be expensive.
 
-<img src="img/new_sql_expectation.png" alt="Save the generated sql expectation" style="width:500px;"/><br>
+<img src="img/generate_new_sql_expectation.png" alt="Generate a custom sql expectation" style="width:500px;"/><br>
```

Comment:
This alt text doesn't really provide information present in the image that's not already available in the surrounding prose. I suggest having the alt text provide the generated SQL query from the screenshot verbatim or having the alt text summarize the info about `{batch}` that's shown in the screenshot.